.github/workflows/ci.yml

```diff
@@ -20,68 +18,61 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v1
 
+    - name: Setting conda path
+      run: echo "::set-env name=PATH::${HOME}/miniconda3/bin:${PATH}"
```

Comment:
Just out of curiosity what is the syntax here? `::` evaulates this?

Reply:
It's just a GitHub actions convention, named `annotations`. The idea is to output that in the job logs, and GitHub is able to understand it and perform an action based on what we output. Another one would be starting a line with `##[error]` which GitHub understands as an error and highlights (and counts) them.

I need to follow up with GitHub to understand it better and see if they have docs, but this one is the syntax to change the `PATH` (or any other environment variable I assume) in a way that persists to next steps of the job.